manifests/init.pp

```diff
@@ -317,6 +319,7 @@
   Boolean $jobs_service_enable = $::foreman::params::jobs_service_enable,
   Boolean $hsts_enabled = $::foreman::params::hsts_enabled,
   Array[Stdlib::HTTPUrl] $cors_domains = $::foreman::params::cors_domains,
+  Stdlib::Port $failed_login_attempts_limit = $::foreman::params::failed_login_attempts_limit,
```

Comment:
This doesn't sound like a TCP port number to me but rather an integer.
```suggestion
  Integer[0] $failed_login_attempts_limit = $::foreman::params::failed_login_attempts_limit,
```

Reply:
True. bad copy/paste from Stdlib::Port $dynflow_pool_size. BTW, shouldn't $dynflow_pool_size also be Integer[0]?

I'll have a look at foreman_config_entry, didn't know it. I had to configure failed_login_attempts_limit and I saw that it was missing, that's why I added :)